nano/secure/ledger.cpp

```diff
@@ -1257,7 +1253,6 @@ void nano::ledger::update_account (nano::write_transaction const & transaction_a
 	}
 	else
 	{
-		store.confirmation_height_del (transaction_a, account_a);
```

Comment:
Why not remove it?

Reply:
There will only be a conf height entry for this account if the open block is cemented, so there should never be a scenario where this account is being rolled back so no need to try delete from conf height table as it shouldn't be there. I will add a `debug_assert` though

Reply:
Confirmation height can be manually cleared I guess

Reply:
Yeh it should delete the conf height entry, good spot!